app/deliver_grant_funding/templates/deliver_grant_funding/grant_list.html

```diff
@@ -1,27 +1,49 @@
 {% extends "common/base.html" %}
 {% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+{% from "govuk_frontend_jinja/components/table/macro.html" import govukTable %}
 
 {% block pageTitle %}
-  Grants - MHCLG Funding Service
+    Grants - MHCLG Funding Service
 {% endblock pageTitle %}
 
 {% set active_item_identifier = "grants" %}
 
 {% block content %}
-  <div class="govuk-grid-row">
+<div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">Grants</h1>
-
-      {{
-        govukButton({
-           "text": "Add a grant",
+        <h1 class="govuk-heading-l">Grants</h1>
+        {{ govukButton({
+           "text": "Set up a grant",
            "href": url_for("deliver_grant_funding.grant_setup_intro")
-        })
-      }}
-
-      {% for grant in grants %}
-        <h2 class="govuk-heading-s"><a href="{{ url_for('deliver_grant_funding.view_grant', grant_id=grant.id) }}" class="govuk-link govuk-link--no-visited-state">{{ grant.name }}</a></h2>
-      {% endfor %}
+        }) }}
+    </div>
+</div>
+<div class="govuk-grid-row">
+    {% if grants %}
+    {% set table_rows = namespace(items=[]) %}
+    {% for grant in grants %}
+    {% set grant_detail_link %}
+    <a class='govuk-link govuk-link--no-visited-state'
+       href={{ url_for("deliver_grant_funding.view_grant", grant_id=grant.id) }}>{{ grant.name }}</a>
+    {% endset %}
+    {% set table_rows.items = table_rows.items + [
+                [{"html": grant_detail_link},
+                 {"text": grant.ggis_number},
```

Comment:
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/b390a24f-67bf-4693-80f2-9a42fcfeaded" />

Can we show an empty field instead of the string `None` here please? I think we're making GGIS number compulsory soon, but for the moment it still looks a bit weird.

```suggestion
                 {"text": grant.ggis_number or ''},
```

Reply:
Fixed